pkg/subctl/cmd/cloud/prepare/aws.go

```diff
@@ -65,16 +56,28 @@ func prepareAws(cmd *cobra.Command, args []string) {
 		},
 	}
 
+	// For load-banalced gateways we want these ports open internally to facilitate private-ip to pivate-ip gateways communications.
+	if gateways == 0 {
+		input.InternalPorts = append(input.InternalPorts,
+			api.PortSpec{Port: nattPort, Protocol: "udp"},
+			api.PortSpec{Port: natDiscoveryPort, Protocol: "udp"},
```

Comment:
```suggestion
			api.PortSpec{Port: natDiscoveryPort, Protocol: "udp"},
			// ESP & AH protocols are used for private-ip to private-ip gateway communications
			{Port: 0, Protocol: "50"},
			{Port: 0, Protocol: "51"},
```

Reply:
Here it's also necessary, if private-ip to private-ip connectivity is possible, and no nat is detected, the gateway will attempt to use ESP/AH

Reply:
these are internal ports between the submariner gateways inside the same cluster, is ESP/AH expected in this scenario? i thought only for gateway-to-gateway external communications

Reply:
ah nvm i see your point